.travis.yml

```diff
@@ -1,33 +1,47 @@
 language: go
 
-services:
-  - docker
+sudo: required
 
 go:
   - 1.9.x
   - master
 
+env:
+  - INTEGRATION=true GOOS=linux
+  - INTEGRATION=false GOOS=linux
+
+deploy:
```

Comment:
Commenting here so it's tracked. 

Like we talked about we should just publish a release to github. Everyone could have different requirements for deploying this to their environments so building a usable docker image for everyone is hard.